src/coreclr/interpreter/compiler.cpp

```diff
@@ -2216,7 +2264,9 @@ void InterpCompiler::CreateLeaveChainIslandBasicBlocks(CORINFO_METHOD_INFO* meth
 
         if (pLeaveChainIslandBB == NULL)
         {
-            pLeaveChainIslandBB = AllocBB(clause.HandlerOffset + clause.HandlerLength);
+            unsigned int bbEHClauseIndex = i + 1;
```

Comment:
A nit - for multiple members of various compiler structs, we use -1 to indicate that the value is not set (e.g. for variable indices, etc). Can we use the same concept here instead of using 0 and then having to add 1 to the index?